webapp.py

```diff
@@ -45,7 +47,11 @@ def predict_from_webcam():
   decoded = decode_base64_img(request.get_json())
   img = get_img_from_decoded(decoded)
 
-  prediction_name, confidence = get_yolo_predictions(yolo_model, img, 0)
+  if model_switch == "yolo":
```

Comment:
If you use the ```request``` to read query params I could add the code to the front end which will send the correct query param back that the user wants to use for model inference.

The url being sent back would look like 
```
localhost:8080/webcam?inference=detectron 
```
or 
```
localhost:8080/webcam?inference=yolo
```

Reply:
<img width="1228" alt="image" src="https://user-images.githubusercontent.com/10001990/204152145-db459349-3aaa-4080-aaa1-fe9d6375ca41.png">